config/routes.rb

```diff
@@ -138,11 +138,18 @@
         get "/#{tab}", to: redirect("/schools/%{school_id}/analysis")
       end
 
-      get 'advice', to: 'advice#index'
-      get 'advice/:key(/:tab)', to: 'advice#show', as: :advice_tab
+      resource :advice, controller: 'advice', only: [:show] do
+        resource :baseload, controller: 'advice/baseload', only: [:show] do
```

Comment:
Just a note for future implementation - if this is a re-useable pattern, the member section *I think* could be defined as a concern.

Reply:
Yes - will try a concern or similar when we introduce more controllers (left it pretty basic in this initial version)